common/src/main/resources/assets/voicechat/lang/en_us.json

```diff
@@ -1,217 +1,212 @@
-  "message.voicechat.mute.disabled": "Press to mute your microphone",
-  "message.voicechat.mute.disabled_ptt": "You can't mute when using push to talk",
-  "message.voicechat.disable.enabled": "Press to enable all voice chat sounds",
-  "message.voicechat.disable.disabled": "Press to disable all voice chat sounds",
-  "message.voicechat.disable.no_speaker": "No speaker available",
-  "message.voicechat.hide_icons.enabled": "Press to show all voice chat icons",
-  "message.voicechat.hide_icons.disabled": "Press to hide all voice chat icons",
-  "message.voicechat.show_group_hud.enabled": "Press to hide the group chat HUD",
-  "message.voicechat.show_group_hud.disabled": "Press to show the group chat HUD",
-  "message.voicechat.recording.disabled": "Press to start recording",
-  "message.voicechat.recording.enabled": "Press to stop recording",
+  "message.voicechat.mute.enabled": "Press to unmute your microphone.",
+  "message.voicechat.mute.disabled": "Press to mute your microphone.",
+  "message.voicechat.mute.disabled_ptt": "You can't mute your microphone when using Push to Talk.",
+  "message.voicechat.disable.enabled": "Press to enable all voice chat sounds.",
+  "message.voicechat.disable.disabled": "Press to disable all voice chat sounds.",
+  "message.voicechat.disable.no_speaker": "No speakers available.",
+  "message.voicechat.hide_icons.enabled": "Press to show all voice chat icons.",
+  "message.voicechat.hide_icons.disabled": "Press to hide all voice chat icons.",
+  "message.voicechat.show_group_hud.enabled": "Press to hide the group chat HUD.",
+  "message.voicechat.show_group_hud.disabled": "Press to show the group chat HUD.",
+  "message.voicechat.recording.disabled": "Press to start recording.",
+  "message.voicechat.recording.enabled": "Press to stop recording.",
   "message.voicechat.no_speak_permission": "You do not have permission to speak",
   "message.voicechat.no_listen_permission": "You do not have permission to hear the voice chat",
   "message.voicechat.no_group_permission": "You do not have permission to join groups",
   "message.voicechat.search_hint": "Search...",
   "message.voicechat.search_empty": "Couldn't find any players with that name",
   "message.voicechat.other_volume": "Other",
-  "message.voicechat.other_volume.description": "The volume of any sound not listed below",
+  "message.voicechat.other_volume.description": "The volume of any sound not listed below.",
   "message.voicechat.more_members": "and %s more...",
   "message.voicechat.no_groups": "No existing groups",
-  "message.voicechat.audio_type": "3D audio: %s",
+  "message.voicechat.audio_type": "3D Audio: %s",
   "message.voicechat.audio_type.normal": "Normal",
   "message.voicechat.audio_type.reduced": "Reduced",
-  "message.voicechat.audio_type.off": "Off",
-  "message.voicechat.invite_player": "Invite %s to your group",
-  "message.voicechat.press_to_reassign_key": "Press to reassign key",
+  "message.voicechat.audio_type.off": "OFF",
```

Comment:
I did it for parity with vanilla Minecraft and other mods. Ever since I can remember, Mojang has always used all caps for "ON" and "OFF", and mod developers have adopted this unwritten rule in their mods to blend in well with the vanilla look and feel.

Reply:
Fair point